app/views/posts/index.html.erb

```diff
@@ -34,6 +34,7 @@
   <% end %>
 
 <button class="pagination">Pagination<button>
+<a href="/users/<%= @current_user.id %>/posts/new" class="btn btn-primary">New Post</a>
 </div>
 
```

Comment:
- [Optional] Dear @dieum512, though you have a working button, currently it is not following the design structure of the rest of the application. Please try to update some styles. 
- [Optional] Also, when ever you have an image in the page please make use of the alternative text, this provides semantic meaning and makes your website SEO wise.

Reply:
Hey @TadesseDev Do you have any idea of how I can make the other change from the first code reviewer? it will be a great help to me

Reply:
Sure @dieum512 
> Please do something like this 
```ruby
    if @comment.save
      redirect_to post_path(@comment.post)
```

Reply:
Thank you :pray: @TadesseDev